### DISCUSSION TITLE
Pair badge not claimed even though co authored commits

### DISCUSSION THREAD

**Discussion by chrisK824** [community/community#53644] [Other Feature Feedback, Questions, & Ideas]
I recently commited some code in a public repository and co authored some of that with another user. The branch was finally merged after a pull request into the main branch, however no badge claimed 😕 

Any ideas why so?

The PR https://github.com/DenverCoder1/github-readme-streak-stats/pull/507

**Comment by javali93** [comment]
I think there should be a commit in the PR that has to be co-authored 

This one below got me the badge

<img width="912" alt="Screenshot 2023-04-25 at 10 52 34 AM" src="https://user-images.githubusercontent.com/26891686/234181715-6e64179e-5d71-4f9a-9aa9-d79ec4fc8d96.png">

**Comment by Ankitz007** [comment]
Hi. 

You need to add co-author in one of your commits in order to successfully obtain that badge. You can do it either via `VS Code` (There is an add Co Authors option) or via `GitHub Desktop`. As you can see in one of the answers, both the authors' profile avatar should display together in a co-authored commit.

**Comment by YasinDehfuli** [comment]
use this course to learn how to get [Pair Extraordinaire](https://github.com/4xmen/Get-Github-Achievements/blob/main/assets/steps/README.pair-extraordinaire.md), badge step by step and by images. it will help you to get this badge. 

Github Repository: 
https://github.com/4xmen/Get-Github-Achievements/blob/main/assets/steps/README.pair-extraordinaire.md

If you need more help to help you get all github achievements use this link : 

https://github.com/4xmen/Get-Github-Achievements
